package.json

```diff
@@ -7,18 +7,17 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint *.js src",
```

Comment:
The eslint config file is not being linted by this script because the config contains strings with double-quotes. Please double check this.